tests/h/streamer/filter_test.py

```diff
@@ -27,114 +32,109 @@ class TestFilterHandler:
             (["http://example.com"], "https://example.com/?", True),
         ],
     )
-    def test_it_matches_uri(self, make_annotation, query_uris, ann_uri, should_match):
-        ann = make_annotation(target_uri=ann_uri)
+    def test_it_matches_uri(
+        self, factories, filter_uris, ann_uri, should_match, filter_matches
+    ):
+        ann = factories.Annotation(target_uri=ann_uri)
 
-        query = {
+        filter_ = {
```

Comment:
We were inconsistently swapping between referring to these as filters and queries. I've standardised on filters. The underscore is to distinguish it from the python built in `filter()`